cores/libretro-net-retropad/net_retropad_core.c

```diff
@@ -586,7 +586,7 @@ static void open_UDP_socket()
    socket_target_t in_target;
 
    if (s && s != SOCKET_ERROR)
-      close(s);
+      socket_close(s);
```

Comment:
This is the only change that looks unrelated, but correct?

Reply:
Same as in https://github.com/libretro/RetroArch/pull/16494/files, forgot to rebase, sorry. But this is already in via #16494 .